types/proxy-from-env/index.d.ts

```diff
@@ -1,13 +1,16 @@
 // Type definitions for proxy-from-env 1.0
 // Project: https://github.com/Rob--W/proxy-from-env#readme
 // Definitions by: JasonHK <https://github.com/JasonHK>
+//                 FloPes <https://github.com/flopes89>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+import { Url } from "url";
```

Comment:
@flopes89 You're missing the triple-slash directive. Please put the code below before line 7.

```typescript
/// <reference types="node" />
```